Makefile

```diff
@@ -896,6 +896,9 @@ CPPLINT_FILES = $(filter-out $(CPPLINT_EXCLUDE), $(wildcard \
 	src/*.c \
 	src/*.cc \
 	src/*.h \
+	src/tracing/.c \
+	src/tracing/.cc \
+	src/tracing/.h \
```

Comment:
You are missing the wildcard character before the file extension (`*.c`).

Also, use `src/*/*.c` instead of `src/tracing/*.c` so that this is future-proof.